Week 1/Homework/calculator.rb

```diff
@@ -0,0 +1,23 @@
+if ARGV.length == 2
+	amount_of_money = ARGV[0].to_f
+	amount_of_months = ARGV[1].to_f
+	interest_rate = rand(5..20).to_f
+	puts "Interest Rate: " + interest_rate.to_i.to_s + "%"
+	[*0..amount_of_months.to_i].each do |i|
+		amount_of_dollars = amount_of_money.to_i
+		amount_of_cents = ((amount_of_money % 1) * 100).to_i
+		amount_of_dollars_string = amount_of_dollars.to_s
+		amount_of_cents_string = amount_of_cents.to_s
+		if amount_of_cents < 10
+			amount_of_cents_string = "0" + amount_of_cents.to_s
+		end
+		time_passed = i.to_s + " months: "
+		if i == 1
+			time_passed = "1 month: "
```

Comment:
Ruby's if statement is a bit more magical, you can actually do the following as well.

```rb
time_passed = "1 month: " if i == 1
```